src/@types/index.d.ts

```diff
@@ -17,3 +17,11 @@ limitations under the License.
 export type Theme = "light" | "light-hc" | "dark" | "dark-hc";
 
 export type Platform = "web" | "android" | "ios";
+
+declare module "prettier" {
+  export interface Options {}
+}
+
+declare module "svgo" {
+  export interface Config {}
+}
```

Comment:
This could do with a comment like that in global.d.ts

Reply:
Han, that's a leftover, I first tried there before moving things to global.d.ts

Reply:
Rebased and removed that